addon/components/overlay-marks.js

```diff
@@ -7,9 +7,13 @@ export default Component.extend({
   layout,
   tagName: '',
   service: service('ember-overlays'),
-  marks: computed('service.marks', function() {
+  marks: computed('service.marks', 'reverseOrder', function() {
```

Comment:
I think `reverseOrder` is not needed anymore here.

Reply:
@ef4 all set.